srcpkgs/gum/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'gum'
+pkgname=gum
+version=0.10.0
+revision=1
+build_style=go
+build_helper=qemu
+go_import_path="github.com/charmbracelet/gum"
+go_ldflags="-X main.Version=v${version}"
+short_desc="Tool for glamorous shell scripts"
+maintainer="zenobit <zen@osowoso.xyz>"
+license="MIT"
+homepage="https://github.com/charmbracelet/gum"
+changelog="https://github.com/charmbracelet/gum/releases"
+distfiles="https://github.com/charmbracelet/gum/archive/refs/tags/v${version}.tar.gz"
+checksum=6084bd89fdb51bbe58ef2a576e81f7d66471a48acc4d8aae41deff5cbb41a2ab
+
+post_install() {
+	vlicense LICENSE
+
+	_cli=${DESTDIR}/usr/bin/gum
+	for shell in bash fish zsh; do
+		vtargetrun $_cli completion $shell > gum.${shell}
+		vcompletion gum.${shell} $shell
+	done
+
+	vtargetrun $_cli man > gum.1
```

Comment:
I don't think there is a reason for this variable, it only makes the code less readable in my opinion.
```suggestion
	for shell in bash fish zsh; do
		vtargetrun ${DESTDIR}/usr/bin/gum completion $shell > gum.${shell}
		vcompletion gum.${shell} $shell
	done

	vtargetrun ${DESTDIR}/usr/bin/gum man > gum.1
```